Manual.md

```diff
@@ -1895,6 +1898,20 @@ Example: `transmission unprivileged user - for uninstalled package transmission`
 
 This trigger can only be used by using the `system_accounts` variable.
 
+<a id="triggers_system_accounts"></a>
+#### texmf-dist
```

Comment:
You can just use an achor for the link, `[texmf-dist](#texmf-dist)`. That way you help with #21609 :p

Reply:
Unless of course there is an argument against moving to anchors.

Reply:
Should I, or not? I feel like this is something that should be standardized, so maybe its better to leave it for a sweeping-change PR...

Reply:
Keep consistent with what is in master.